src/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -4,14 +4,19 @@ import {
 	Input,
 	Output,
 	EventEmitter,
-	Optional
+	Optional,
+	ChangeDetectionStrategy
 } from "@angular/core";
 import { DomSanitizer } from "@angular/platform-browser";
 import { Router } from "@angular/router";
 
 @Component({
 	selector: "cds-breadcrumb-item, ibm-breadcrumb-item",
 	template: `
+		<ng-template #content>
+  		<ng-content/>
+		</ng-template>
+
```

Comment:
What is the reason for this? 🤔

Reply:
Whoops, nevermind, I realized the issue in the new syntax migration. Thanks for fixing this!